test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlDataContractSerializerOutputFormatterTest.cs

```diff
@@ -626,6 +628,58 @@ public void GetSupportedContentTypes_ReturnsSupportedTypes(Type type, object exp
             XmlAssert.Equal(expectedOutput, content);
         }
 
+        public static IEnumerable<object[]> LogsWhenUnableToCreateSerializerForTypeData
```

Comment:
Would be slightly easier to understand using `TheoryData<XmlDataContractSerializerOutputFormatter, TestSink>`.